boltdb/boltdb.go

```diff
@@ -144,6 +144,17 @@ func (db *TaskDB) DeleteTask(id string) error {
 // ListNonTerminalTasks returns a list of tasks that are not yet finished in one
 // way or another.
 func (db *TaskDB) ListNonTerminalTasks() ([]*eremetic.Task, error) {
```

Comment:
Is this used directly anywhere now or could we drop it?

Reply:
its used in reconciliation, but yeah we can call directly to ListTasks with right filter.

Reply:
sounds good, better to rip out the legacy function in one go